docs/pages/includes/tctl.mdx

```diff
@@ -1,35 +1,28 @@
-Make sure you can connect to Teleport. Log in to your cluster using `tsh`, then use `tctl`
-remotely:
+To check that you can connect to your Teleport cluster, sign in with `tsh`, then 
+run `tctl` commands remotely:
 
 {/* Ignoring scope linting since we use this partial throughout the docs and
 cannot guarantee that it will line up with a page's configured scopes*/}
 {/*lint ignore scopes*/} 
-<ScopedBlock scope={["oss", "enterprise"]}>
 
 ```code
-$ tsh login --proxy=teleport.example.com --user=email@example.com
+$ tsh login --proxy=<Var name="teleport.example.com" /> --user=<Var name="email@example.com" />
 $ tctl status
-# Cluster  teleport.example.com
+# Cluster  (=teleport.url=)
 # Version  (=teleport.version=)
 # CA pin   (=presets.ca_pin=)
 ```
 
-You can run subsequent `tctl` commands in this guide on your local machine.
-
-For full privileges, you can also run `tctl` commands on your Auth Service host.
+<ScopedBlock scope={["oss", "enterprise"]}>
+If you can connect to the cluster and run the `tctl` command remotely,
```

Comment:
I'm not sure what you mean by this. If you've done a `tsh login` then you would be running tctl on your local computer. What does "run the tctl command remotely" mean in this context?

I _think_ we probably want to simplify and remove the words _local_ and _remote_. Maybe refer to "local computer" as "your workstation" instead?

Reply:
@zmb3 I'm still trying to figure out how this `tsh login` followed by running `tctl` commands (that appears in pretty much every topic) should work.
When I enroll a server as a resource, I can run `tsh login --proxy=proxy-address --user=admin-user`, authenticate as this user, then run `tsh` commands on the local server. I can't run any `tctl` commands or `sudo tctl` commands on the enrolled server. I can only run `tctl` commands on the Auth service host. I've tried creating an identity file and using the `--auth-server` and `--identity` flags to run `tctl` commands on the enrolled server, but I haven't found the right combination of command-line options to make it work.

```
ubuntu@ip-192-168-13-57:~$ tctl status
ERROR: failed to open file for Teleport configuration: /etc/teleport.yaml. Ensure that you are running as a user with appropriate permissions.
open /etc/teleport.yaml: permission denied

ubuntu@ip-192-168-13-57:~$ sudo tctl status
ERROR: tctl must be either used on the auth server or provided with the identity file via --identity flag
```

Is it possible to run `tctl` commands on a non-auth-server? what's the magic incantation that allows it to happen? (I found this page https://goteleport.com/docs/reference/cli/#tctl, but I must be missing something that is obvious to others.)